index.bs

```diff
@@ -2411,15 +2410,16 @@ additional information about a device,
 or device identifiers,
 each increase the risk of harming the user's privacy.
 
-One risk is that as more specific information is shared,
+The web app should not be able to distinguish between the user rejecting
```

Comment:
```suggestion
A web app should not be able to distinguish between the user rejecting
```

Do we have guidance around the usage of terms like "web app" and "[web]site"?  I have something of a strong preference for the latter, but acknowledge that there might be some sort of significant difference.